src/github.com/matrix-org/dendron/logging/hook.go

```diff
@@ -0,0 +1,92 @@
+package logging
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"os"
+	"sync/atomic"
+)
+
+// NewFSHook makes a logging hook that writes JSON formatted
+// log entries to info, warn and error log files.
```

Comment:
Worth writing here what the contents of each file will be (i.e. that they contain error messages of that severity *or more severe*, rather than just separated by level)